docs/app/client/carousel/examples/SelectIndex.tsx

```diff
@@ -22,8 +22,14 @@ const SelectIndex = () => {
           </Chip>
         ))}
       </div>
-      <Carousel step={1} selectedIndex={selected} className="w-[320px]">
-        <Carousel.Reel className="overflow-x-hidden">
+      <Carousel
+        step={1}
+        selectedIndex={selected}
+        className="w-[320px]"
+        data-testid="selectedIndex-carousel"
+        isSwipeDragDisabled
```

Comment:
Adding the attribute to an implementation is a potential breaking change. Can it be done differently?

Reply:
Yes, removing.